README.md

```diff
@@ -119,30 +115,78 @@ Marie Curie
 - `USE_STRICT_ACCESS_KEYS` -> if set to `false` it generates the same access code for all moderators, otherwise it generates 1 code per moderator
 - `DOMAIN_PREFIX` -> the domain where the application will be hosted.
 
-#### Run
-
-- In order to run, just use the TEST method in lambda, with the test event created above.
+Following screenshot shows the **TEST** event UI used to generate the resources. (It can be hard to know what is meant by the test event)
+![generateLiveEventLambdaTrigger](https://github.com/aws-samples/amazon-chime-live-events/blob/master/resources/generate_live_event_lambda_trigger.png)
```

Comment:
I dont see this picture?

Reply:
It's because the url is pointing to the master branch tree and the images have not been merged into master yet. I noticed they were pointing to update-documentation branch but that is not correct so I updated the urls.